zipline/gens/sim_engine.pyx

```diff
@@ -86,7 +83,7 @@ cdef class MinuteSimulationClock:
                     yield minute, MINUTE_END
 
             if not minute_emission:
-                yield day, DAY_END
+                yield minutes[-1], DAY_END
```

Comment:
This is  the minute mode clock. In both daily and minute emissions, the prices used for the day end should be at the market_close.

Noticed this when using `get_loc` instead of a combination of searchsorted for getting the minute position. From reading the code it may have been that the first minute of the day was being used for the last sale prices for risk calculations, but have not confirmed.